### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by OmerFlame** [kodika/MutataApp#10]
I just want to know if it's dead or not.

**Comment by kostassite** [comment]
Hi @OmerFlame 
It is alive and we are going to release a new version in the next 5-10 days with lots of new frameworks

**Comment by OmerFlame** [comment]
Is this new version going to fix the issues that I reported to you in the past?

**Comment by kostassite** [comment]
Yeah, most of the reported issues should be fixed by now. I will close the issues when the new release is uploaded
